### PR TITLE
Add hcloud network info

### PR DESCRIPTION
**Pull request by ghost**
##### SUMMARY

Add hcloud_network_info
##### ISSUE TYPE

- New Module Pull Request

##### ADDITIONAL INFORMATION
hcloud_network_info

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/hcloud/hcloud_network_info.py [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+            returned: always
+            type: int
+            sample: 1937415
+        name:
+            description: Name of the network
+            returned: always
+            type: str
+            sample: awsome-network
+        ip_range:
+            description: Ip range of the network
```
Comment (LKaemmerling) [inline lib/ansible/modules/cloud/hcloud/hcloud_network_info.py]:
```suggestion
            description: IP range of the network

File: lib/ansible/modules/cloud/hcloud/hcloud_network_info.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+            returned: always
+            type: str
+            sample: 10.0.0.0/16
+        subnetworks:
+            description: subnetworks belonging to the network
```
Comment (LKaemmerling) [inline lib/ansible/modules/cloud/hcloud/hcloud_network_info.py]:
```suggestion
            description: Subnetworks belonging to the network

**Review by LKaemmerling — CHANGES_REQUESTED** [review]

**Comment by LKaemmerling** [comment]
shipit

**Review by felixfontein — APPROVED** [review]
shipit

**Comment by felixfontein** [comment]
rebuild_merge

**Comment by felixfontein** [comment]
@cschmitt-hcloud @LKaemmerling thanks for adding this new module!

**Pull request merged**